src/analyzing/tests/Test_analyze_generic.ml

```diff
@@ -64,9 +64,8 @@ let test_cfg_il ~parse_program file =
   Naming_AST.resolve lang ast;
   Visit_function_defs.visit
     (fun _ fdef ->
-      let _, xs = AST_to_IL.function_definition lang fdef in
-      let cfg = CFG_build.cfg_of_stmts xs in
-      Display_IL.display_cfg cfg)
+      match CFG_build.cfg_of_fdef lang fdef with
+      | { fparams = _; fcfg } -> Display_IL.display_cfg fcfg)
```

Comment:
No need for a `match`, a `let` is more clear when there is _just one_ case to match.

Reply:
I never knew about this pattern, so this nit picking is very useful!